chapters/en/chapter2/tts_pipeline.mdx

```diff
@@ -0,0 +1,98 @@
+seems more descriptive for your task. 
+
+Let's explore how you can use this pipeline to start generating audio narration for texts, and music with just a few lines of code.
+
+This pipeline is new to 🤗 Transformers, thus you'll need to install the library from the source: 
+
+```bash
+pip install git+https://github.com/huggingface/transformers.git
+```
+
+## Generating speech
+
+Let's begin by exploring text-to-speech generation. First, just as it was the case with audio classification and automatic 
+speech recognition, we'll need to define the pipeline. We'll use the `suno/bark-small` model with this pipeline:
+
+```python
+from transformers import pipeline
+
+pipe = pipeline("text-to-speech", model="suno/bark-small")
+```
+
+The next step is as simple as passing some text through the pipeline. All the preprocessing will be done for us under the hood: 
+
+```python
+text = "Ladybugs have had important roles in culture and religion, being associated with luck, love, fertility and prophecy. "
+output = pipe(text)
+```
+
+In a notebook, we can use the following code snippet to listen to the result: 
+
+```python
+from IPython.display import Audio
+
+Audio(output["audio"], rate=output["sampling_rate"])
+```
+
+The model that we're using with the pipeline, Bark, is actually multilingual, so we can easily substitute the initial 
+text with a text in, say, French, and use the pipeline in the exact same way. It will pick up on the language all by itself:
+
+```python
+fr_text = "Contrairement à une idée répandue, le nombre de points sur les élytres d'une coccinelle ne correspond pas à son âge, ni en nombre d'années, ni en nombre de mois. "
+output = pipe(fr_text)
+Audio(output["audio"], rate=output["sampling_rate"])
+```
+
+Not only is this model multilingual, it can also generate audio with non-verbal communications and singing. Here's how 
+you can make it sing: 
+
+```python
+song = "♪ In the jungle, the mighty jungle, the ladybug was seen. ♪ "
+output = pipe(song)
+Audio(output["audio"], rate=output["sampling_rate"])
+```
+
+We'll dive deeper into Bark specifics in the later unit dedicated to Text-to-speech, and will also show how you can use 
+other models for this task. Now, let's generate some music!
+
+## Generating music
+
+Just as before, we'll begin by instantiating a pipeline. For music generation, we'll take the pretrained `facebook/musicgen-small` 
+checkpoint.  
+
+```python
+music_pipe = pipeline("text-to-audio", model="facebook/musicgen-small")
+```
+
+Let's create a text description of the music we'd like to generate:
+
+```python
+text = "90s rock song with electric guitar and heavy drums"
+```
+
+For best results, we'll specify some additional music generation parameters to pass to `musicgen`. These are model-, and 
+not pipeline-specific. 
+
+- `do_sample` introduces some variability and a bit of randomness to improve the "creativeness" of the output
```

Comment:
Sampling and guidance scale should be set by default in the checkpoint's generation config: https://huggingface.co/facebook/musicgen-small/blob/51027f0bee8489c1750a7b8a4806894ab2e7dc4d/generation_config.json#L5-L6

=> so we can probably remove them to simplify the code example? The default values in the generation config are the 'optimal' values that they found in the paper, so they generate pretty good results on average

Changing the max new tokens is helpful though to get longer generations!

Reply:
Agreed, left only the max new tokens parameter.